.github/workflows/backend_node_actions.yml

```diff
@@ -26,20 +26,24 @@ jobs:
     defaults:
       run:
         shell: bash
-        working-directory: ./backend
-        
+
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [13.x]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm i
-    - run: npm run build --if-present
-    - run: npm run test:unit
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
```

Comment:
Just for some clarity on this (not a comment about code validity or anything), so basically, it says: use actions/cache lib.
And with: is actually more like options: ? 
Then we have the path which is npm, little bit lost here? 
Key simply hashes the plock.
restore-keys grabs runner.os ? I'm lost again lol!

Reply:
Apparently like you said, you aren't meant to cache node_modules with npm.

https://github.com/actions/cache/blob/master/examples.md#node---npm

I've removed the caching step since it isn't faster and doesn't play well with `npm ci`.